apps/ai-content-generator/src/components/app/sidebar/feature-button/FeatureButton.tsx

```diff
@@ -19,7 +19,7 @@ const FeatureButton = (props: Props) => {
   const handleOnClick = () => {
     const entryId = sdk.entry.getSys().id;
 
-    const dialogConfig = makeDialogConfig({ feature, entryId }, dialogTitle);
+    const dialogConfig = makeDialogConfig({ feature, entryId });
```

Comment:
Since we're just using the feature we no longer need the dialogTitle prop in this component